pkg/rancher-desktop/backend/containerClient/mobyClient.ts

```diff
@@ -474,11 +474,13 @@ export class MobyClient implements ContainerEngineClient {
     const isCLIPlugin = /^docker-(?!credential-)/.test(executableName);
     const binType = isCLIPlugin ? 'docker-cli-plugins' : 'bin';
     const binDir = path.join(paths.resources, process.platform, binType);
```

Comment:
Doesn't this have to be hard-coded to `bin` now? Otherwise you will just add the plugin dir twice.

```suggestion
    const binDir = path.join(paths.resources, process.platform, 'bin');
```

I find the code a bit confusing to read because `binDir` and `executableDir` don't really make it clear which one is the plugin dir. Shouldn't it be something like this (pseudocode):

```
dirsToAdd = [binDir]
if isCLIPlugin {
  dirsToAdd.push(pluginDir)
}
```

So if it isn't a plugin then you add only a single directory.

Reply:
Good catch. Rewrote this to be hopefully clearer.

(It was using a `Set` before so there weren't duplicates, but it's probably clearer to be explicit.)